config/prometheus/alert-rules.yml

```diff
@@ -9,13 +9,21 @@ groups:
         annotations:
           error: "Montagu on {{ $labels.instance }} is down"
 
-  - name: orderly-web
+  - name: montagu-packit
```

Comment:
I wouldn't mind these alerts targeting all packit-apis, including the non-montagu ones. See my comments in `prometheus.yml` about using `job="packit-api"`.

(I definitely think the alert name and message should reference the "API" word)